.gitignore

```diff
@@ -1,5 +1,20 @@
-.DS_Store
```

Comment:
Can we re-add .DS_Store? It's very useful. It's for the files that OSX auto generates every time you view a folder in finder.